### DISCUSSION TITLE
솔루션 심사 요청 - 테스트 계정 관련 문의

### DISCUSSION THREAD

**Discussion by dminwoo** [commerce-api-naver/commerce-solutions#95] [묻고 답하기]
안녕하세요,

현재 커머스API센터에서 솔루션을 등록하여, 테스트까지는 끝마치고 솔루션 심사 요청을 준비하고 있는 단계입니다.

심사 요청 사항 중 아래의 내용에 대해 궁금증이 생겨 질문글 올립니다.
```
- 테스트 계정 ID / PW

- 구독 테스트 방법

- 해지 테스트 방법
```

우선 구독 흐름이,
1. 솔루션마켓에서 우리 솔루션을 찾아서 추가하기
2. 그럼 우리 회사의 서비스(페이지)로 이동해서 계정 연동 겸 회원 가입하기
3. 솔루션마켓에서 구독한 내용이, 우리 회사 서비스에도 연동이 되어서 사용 가능한 상태가 됨.

이런 흐름인데요, 따라서 제가 테스트를 진행할때도
저의 아이디(회사 아이디)로 네이버커머스 가입(및 스스 가입 등)을 한 아이디를 통해서,
솔루션을 추가하고, 실제 우리 서비스 홈페이지로 가서 회원가입 및 연동 테스트를 진행했습니다.

예를들어, 네이버커머스에 가입한 아이디는 official@naver.com 이고, 
이를통해 저희 회사에서는 official 이란 계정을 만드는 식 입니다.

구독 테스트와 해지 테스트는 위의 계정으로 진행하는 방식을 글로 쓰고, 영상으로 찍어서 보내드리면 될거 같은데

**궁금한건, 심사 요청 사항에 적는 테스트 계정은
위의 offical@naver.com 또는 official 계정 정보를 알려드리면 되는 것인지?
아니면 아예 네이버커머스에서 테스트 용도로 가입을 하거나, 받아서 테스트 해볼 수 있도록 하는 방법이 있는것인지 궁금합니다.**

**Comment by commerce-api-naver** [comment]
안녕하세요. 커머스API 이민석입니다.

먼저 이 공간은 기술문의에 대한 답변만 가능하기에 심사나 운용 정책 등의 정확한 답변은 솔루션을 관리하는 커머스ID로 [스마트스토어 고객센터](https://help.sell.smartstore.naver.com)에 문의하시어 크로스체크하시는 것을 권해드립니다.
- '1:1 문의하기' 내 상담유형 [커머스솔루션 > 커머스솔루션마켓 입점심사] 선택 권장

통상 솔루션 심사는 말씀하신 내용을 포함한 일련의 솔루션 구독/해지/사용 흐름에서 연동상의 이상 여부 등을 점검하게 됩니다.
따라서 심사 과정에서 필요한 테스트 계정은 네이버 사이드가 아닌 솔루션 시스템 사이드의 계정 정보가 필요할 수 있습니다.
- 참고 - 솔루션 심사와 관련한 셀프 체크리스트: https://apicenter.commerce.naver.com/ko/basic/solution-doc#section/%EC%B6%9C%EC%8B%9C%ED%95%98%EA%B8%B0

> [!NOTE]
> 네이버에서는 어떠한 경우에도 네이버 계정 혹은 커머스ID 계정 정보를 요구하지 않습니다.
> 보유하고 계신 네이버/커머스ID 계정 정보를 제 3자에게 제공하거나 공유되지 않도록 각별히 주의해주시기 바랍니다.

**Reply by dminwoo** [reply]
'심사 과정에서 필요한 테스트 계정은 네이버 사이드가 아닌 솔루션 시스템 사이드의 계정 정보가 필요할 수 있습니다.' + 네이버에서 직접 계정 정보를 요구할 일이 없겠군요 
빠른 답변 감사드립니다 !
